### ISSUE TITLE
Godot Engine Project Manager windows turns completely gray and a bit distorted on startup

### ISSUE THREAD

**Issue by SeychellesDev** [godotengine/godot#57977]
### Godot version

3.4.2-stable

### System information

Windows 11, RTX 3070, OpenGL ES 3.0, Geforce Game Ready Driver v511.65

### Issue description

Upon trying to launch godot engine, a cmd prompt and the godot engine project manager window opens up. Already a visual artifact shows up on the project manager window as a black square to the left and center of window as shown in screenshot.
![image](https://user-images.githubusercontent.com/52707287/153620825-1f248a25-3ba0-4dad-97ce-0cb89eeba51a.png)

When I try to click Open Asset LIbrary or Cancel or the Close button, godot turns into this.
![image](https://user-images.githubusercontent.com/52707287/153620999-104aad30-1f0a-4da7-b42d-88616b72209f.png)

This also leaves weird artifacts on other parts of my screen
![image](https://user-images.githubusercontent.com/52707287/153621092-31fceee9-5d87-44d2-92f6-9e13e27a0ad0.png)

This makes me unable to use the godot engine

### Steps to reproduce

Run Godot.exe file
Try to press any buttons in the project manager window, or switch to another program (alt + tab)
Godot will now have a grey screen everywhere, but can notice slightly that the buttons still work in the background

**Comment by mhilbrunner** [comment]
Does this occur every time? (I assume so, but just to make sure)
Does this occur on earlier Godot versions, i.e. 3.4.1 or more interestingly 3.3.x?

**Comment by SeychellesDev** [comment]
Occurs every time, I have tried the following versions:
3.4.1 (64-bit)
3.3.4 (64-bit)
3.3 (64-bit)
3.3.1 (64-bit)

**Comment by Calinou** [comment]
@SeychellesDev Are you using a laptop with hybrid graphics (NVIDIA Optimus)?

**Comment by SeychellesDev** [comment]
I am using a desktop pc, with an Asus RTX 3070 Dual GPU, connected to 2 monitors, 1 via hdmi and 1 via display port. 
Upon running godot engine with --video-driver GLES2, instead of the grey screen I get a Godot Engine Logo when I try to press a button, and nothing happens.
![image](https://user-images.githubusercontent.com/52707287/153699198-6214c32d-26c9-4059-b11c-8679522d94c3.png)

![image](https://user-images.githubusercontent.com/52707287/153699177-e584009f-f3c7-4402-831c-7d88f9784b24.png)

**Comment by SeychellesDev** [comment]
So after encountering the same issue with VoxEdit program, I realised it may be my system causing the error. I have an asus Prime X570 motherboard, and it installed some motherboard software when setting up my pc. This had caused issues with my windows before, although not similar. So I decided to uninstall anything to do with asus motherboard software, (Armoury Crate, RGB programs etc) and now Godot as well as VoxEdit works fine!

TLDR: Anyone having this same issue try uninstalling the ASUS programs that gets installed when you first setup your pc with a new motherboard. This fixed it for me! 

Thank you all to those who tried to help.
